src/Clusterer.php

```diff
@@ -105,7 +105,8 @@ public function setSaveCoordinates($save)
      */
     public function setMinClusterLocations($limit)
     {
-        $this->minLocations = $limit;
+        // simple sanity check. It doesn't make sense to have clusters with less than 2 locations
+        $this->minLocations = (int)($limit > 2 ? $limit : 0); 
```

Comment:
Currently I working on a script to automatically set the number of minLocations based on the zoom level and on the number of markers on the map. Sometimes this script provides a small number between 0 and 2. This sanity check is being done in this script, however I believe it should be done in the library in order to avoid problems. Throwing an exception in this case (and in my opinion) is not advised, due to that it is more appropriate to just change it. Also adding an exception would cause an incompatibility with older versions, requiring developers to adjust their code.
I would only throw an exception if debug mode was on....